src/components/surveys/survey-design/question-edit/rhs-subcontrols/Year.test.tsx

```diff
@@ -46,6 +46,7 @@ function setUp(step: YearQuestion) {
 
 const setUpWithChangeMock = (step: YearQuestion) => {
   const onChangeMock = jest.fn()
+  jest.useFakeTimers()
```

Comment:
Update this set up method to explicitly use fake timers, rather than depending on a previous test to set fake timers.